superset/connectors/base.py

```diff
@@ -0,0 +1,158 @@
+import json
+
+from sqlalchemy import Column, Integer, String, Text, Boolean
+
+from superset import utils
+from superset.models.helpers import AuditMixinNullable, ImportMixin
+
+
+class BaseDatasource(AuditMixinNullable, ImportMixin):
+
+    """A common interface to objects that are queryable (tables and datasources)"""
```

Comment:
Personally I'm ok with up to 85-90 and that's the way I set up pylint / landscape but I can't do it on codeclimate...